### PR TITLE
test: various cucumber fixes

### PR DESCRIPTION
**Pull request by stringhandler**
Description
---

* improved reorg test to ensure max difficulty
* reduced various warn logs to info or debug
* allowed dns_seeds to be passed as a comma separated string when passed as an ENV var

Motivation and Context
---
The Mass Multiple Reorg cucumber test was flaky because it did not use explicit difficulties in the chains, so it had vague statements "All chains are on the same tip"

How Has This Been Tested?
---
Tested locally

### CODE REVIEW

**Review by sdbondi — DISMISSED** [review]
Nice

File: integration_tests/features/support/steps.js [hunk 1]
```diff
@@ -250,12 +250,18 @@ Given(
...
       async () => {
-        let node_a_result = (await nodeA.get_node_state()) === "LISTENING";
-        let node_b_result = (await nodeB.get_node_state()) === "LISTENING";
+        let nodeAState = await nodeA.get_node_state();
+        console.log(`${nodeNameA} state: ${nodeAState}`);
+        let nodeBState = await nodeB.get_node_state();
+        console.log(`${nodeNameB} state: ${nodeBState}`);
+        let node_a_result = nodeAState === "LISTENING";
+        let node_b_result = nodeAState === "LISTENING";
         return node_a_result && node_b_result;
```
Comment (hansieodendaal) [inline integration_tests/features/support/steps.js]: I suggest not having these checks in here. It is already tested for in `baseNodeProcess.js/start -> baseNodeProcess.js/run` and includes other variants of possible console output as well.

I found that these additional checks let this step time out, possibly due to `await this.client.GetTipInfo().sendMessage({});` not returning directly after base node startup.

File: integration_tests/features/Reorgs.feature [hunk 2]
```diff
@@ -140,23 +140,23 @@ Feature: Reorgs
   Scenario Outline: Massive multiple reorg
```
Comment (hansieodendaal) [inline integration_tests/features/Reorgs.feature:140]: Nicely predictable

File: integration_tests/features/support/steps.js [hunk 3]
```diff
@@ -1316,13 +1312,31 @@ When(
   }
 );
 
+When(
+  /I mine (.*) blocks on (.*) with difficulty (.*)/,
+  { timeout: 20 * 1000 },
```
Comment (hansieodendaal) [inline integration_tests/features/support/steps.js:1335]: This time out may be a little low if we mine many blocks. Suggest 80s.

**Review by hansieodendaal — COMMENTED** [review]
Looks good, just two suggestions to consider.

File: integration_tests/features/support/steps.js (hunk 1 above)
Comment (stringhandler) [reply integration_tests/features/support/steps.js]: Thanks I see that now

**Review by philipr-za — APPROVED** [review]
LGTM

File: integration_tests/features/support/steps.js [hunk 4]
```diff
@@ -1364,11 +1351,14 @@ When(
   /mining node (.*) mines (\d+) blocks$/,
   { timeout: 1200 * 1000 }, // Must allow many blocks to be mined; dynamic time out below limits actual time
   async function (miner, numBlocks) {
-    const miningNode = this.getMiningNode(miner);
+    const miningNode = this.getMiningNode(miner);d
```
Comment (delta1) [inline integration_tests/features/support/steps.js:1354]: this won't parse

Comment (stringhandler) [reply integration_tests/features/support/steps.js:1354]: lol, thought I fixed that

**Pull request merged**